code/modules/jobs/job_types/engineering.dm

```diff
@@ -77,7 +77,7 @@ Station Engineer
 	access = list(access_engine, access_engine_equip, access_tech_storage, access_maint_tunnels,
```

Comment:
can i just say, some of these access changes seem to be a bit drastic, cook is meant to have morgue access to mince bodies, which if we are trying to justify this with how a space station should work, is what would happen since recycling bodies into food, sci would need tech access since they can use some of the boards in there and no one really uses it as is.